examples/basic/annotations/whisker.py

```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between car type and highway MPG from the autompg
+sample data.  This example demonstrates the use of whiskers to display quantile ranges in the plot.
```

Comment:
```suggestion
sample data. This example demonstrates the use of whiskers to display quantile ranges in the plot.
```

Here is an unnecessary whitespace.

Reply:
Ahh good catch @mosc9575 ! @Faith-Nchifor please make this last edit, and *then* I will merge :D

Reply:
Thank you @mosc9575 I never even noticed!